x-pack/test/monitoring_api_integration/packages.ts

```diff
@@ -11,6 +11,7 @@ const PACKAGES = [
   { name: 'beat', version: '0.0.1' },
   { name: 'elasticsearch', version: '1.2.0' },
```

Comment:
Unrelated to this PR but do we need to update this package based on it now being version 1.4?

Reply:
I'll take the opportunity to bump the versions